interop/src/grpc_testing_reconnect_service_client.erl

```diff
@@ -10,7 +10,7 @@
 -compile(export_all).
 -compile(nowarn_export_all).
 
--include_lib("grpcbox/include/grpcbox.hrl").
+-include_lib("include/grpcbox.hrl").
```

Comment:
These should stay `grpcbox/include/gprcbox.hrl`.

Reply:
No wait, this isn't a separate app.

If these also work with `-include("grpcbox.hrl").` then I'd accept that.